SwanProjects/src/ProjectDialog.tsx

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) SWAN Team.
+// Author: Omar Zapata CERN 2021
+
```

Comment:
Same comment as before (and valid for all the rest of typescript files here), it would be good to document that is in each file.